### DISCUSSION TITLE
image seen in local network but not showing up on pages

### DISCUSSION THREAD

**Discussion by alyssxkim** [community/community#146903] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hi! i'm relatively new to working with github pages, and i'm running into an issue related to displaying images. i can see the images i'm trying to use on my local network (npm run dev), but, once i try to see it on my github pages (npm run deploy), the images won't pop up. i've tried all the different fixes i could find (making sure capitalizations are correct, putting the images in different folders, etc.), but nothing seems to work. what could be the issue?

**Comment by Halliday202** [comment]
You're supposed to wait for the whole thing to build.Dont know if this helps
